src/NewTwitchApi/Resources/EventSubApi.php

```diff
@@ -0,0 +1,275 @@
+    {
+        return $this->subscribe(
+            $bearer,
+            $secret,
+            $callback,
+            'channel.subscribe',
+            '1',
+            ['broadcaster_user_id' => $twitchId],
+        );
+    }
+
+    /**
+     * @link https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types#channelcheer
+     */
+    public function subscribeToChannelCheer(string $bearer, string $secret, string $callback, string $twitchId): ResponseInterface
+    {
+        return $this->subscribe(
+            $bearer,
+            $secret,
+            $callback,
+            'channel.cheer',
+            '1',
+            ['broadcaster_user_id' => $twitchId],
+        );
+    }
+
+    /**
+     * @link https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types#channelraid
+     */
+    public function subscribeToChannelRaid(string $bearer, string $secret, string $callback, string $twitchId): ResponseInterface
+    {
+        return $this->subscribe(
+            $bearer,
+            $secret,
+            $callback,
+            'channel.raid',
+            '1',
+            ['broadcaster_user_id' => $twitchId],
+        );
+    }
+
+    /**
+     * @link https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types#channelban
+     */
+    public function subscribeToChannelBan(string $bearer, string $secret, string $callback, string $twitchId): ResponseInterface
+    {
+        return $this->subscribe(
+            $bearer,
+            $secret,
+            $callback,
+            'channel.ban',
+            '1',
+            ['broadcaster_user_id' => $twitchId],
+        );
+    }
+
+    /**
+     * @link https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types#channelunban
+     */
+    public function subscribeToChannelUnban(string $bearer, string $secret, string $callback, string $twitchId): ResponseInterface
+    {
+        return $this->subscribe(
+            $bearer,
+            $secret,
+            $callback,
+            'channel.unban',
+            '1',
+            ['to_broadcaster_user_id' => $twitchId],
+        );
+    }
+
+    private function subscribeToChannelModerator(string $bearer, string $secret, string $callback, string $twitchId, string $eventType): ResponseInterface
+    {
+        return $this->subscribe(
+            $bearer,
+            $secret,
+            $callback,
+            sprintf('channel.moderator.%s', $eventType),
+            '1',
+            ['broadcaster_user_id' => $twitchId],
+        );
+    }
+
+    /**
+     * @link https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types#channelmoderatoradd
+     */
+    public function subscribeToChannelModeratorAdd(string $bearer, string $secret, string $callback, string $twitchId): ResponseInterface
+    {
+        $this->subscribeToChannelModerator($twitchId, 'add', $callback, $bearer, $secret);
```

Comment:
Need to update this to match the new `subscribeToChannelModerator` variable syntax